pkg/azurediskplugin/Dockerfile

```diff
@@ -12,13 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM alpine:3.14.1
-RUN apk add --no-cache util-linux e2fsprogs e2fsprogs-extra ca-certificates udev xfsprogs xfsprogs-extra
+FROM k8s.gcr.io/build-image/debian-base:bullseye-v1.0.0
```

Comment:
I'd prefer to stick with the alpine-based image as it is *much* smaller than Debian.

Reply:
Using this file only for v1 in the Makefile. Should that be fine?

Reply:
See my [previous comment](https://github.com/abhisheksinghbaghel/azuredisk-csi-driver/pull/205#discussion_r719526284).